.flake8/azul_flake8.py

```diff
@@ -30,6 +29,10 @@
     config,
 )
 
+sys.path.append(f'{config.project_root}/scripts/')
```

Comment:
I don't understand why this solves the problem.

Reply:
Added a comment